### ISSUE TITLE
Put an AIS antenna on the roof

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#1481]
While there is scaffolding up for the roof work, John C is planning to install a LoRaWAN gateway on the roof (and has permission from the landlord).

I'm wondering if we should put an AIS antenna up there too, so we could [set up a Raspberry Pi to decode the ship traffic](https://www.rtl-sdr.com/rtl-sdr-tutorial-cheap-ais-ship-tracking/) on the Mersey

**Comment by johnmckerrell** [comment]
Yes, and Flight Radar/ADSB Exchange too. I haven’t investigated AIS myself but the planes are really easy, both Flight Radar and ADSB Exchange give easy scripts (and even a Pi image). TBH I don’t know if that would require a second SDR and antenna or if AIS and ADS can run over the same hardware.

**Comment by amcewen** [comment]
AIS is 162MHz.  I've not looked at the plane stuff at all.  We can probably ask @kayharvey for advice on any antenna issues if you find out the frequencies it uses

**Comment by johnmckerrell** [comment]
Ah, ADSB is 1090MHz so I guess not. Might be worth running some tests to see if we'd actually be able to receive the traffic from the river where we are, could still be plenty of buildings blocking the path?

**Comment by amcewen** [comment]
Have ordered an [antenna](https://www.nevadaradio.co.uk/product/sirio-sb-1-s/) and an [SDR module](https://www.rtl-sdr.com/buy-rtl-sdr-dvb-t-dongles/) to try it out.  Probably not all going to get here before the scaffolding comes down though :-/

**Comment by amcewen** [comment]
The antenna and module have turned up now.  Handily @kayharvey had a suitable cable because the pigtails I bought didn't have quite the right connectors.  Now to get it up and running...
